docs/source/usage/usage.rst

```diff
@@ -67,11 +67,7 @@ Certain information is required to create a new Vendor, make sure you know these
 *Description*
 	A brief description of the business.
 *Address*
-	The street address where the products are being sold.
-		* Street Address
-		* City
-		* State
-		* Zipcode
+	The address field where address of the vendor is submitted, this address is broken into strret, city , state, zip and is sent to the database.
```

Comment:
Should be two sentences (or a semicolon) after ``the vendor is submitted``.

also, ``street``.

And instead of saying "is sent to the database", I'd say "is stored as" the various parts.